gokart/task.py

```diff
@@ -297,6 +297,10 @@ def _make_hash_id(self):
         def _to_str_params(task):
             if isinstance(task, TaskOnKart):
                 return str(task.make_unique_id()) if task.significant else None
+
+            if not isinstance(task, luigi.Task):
+                raise ValueError(f"Task.requires method returns {type(task)}. You should return some kind of GokartTask.")
```

Comment:
```suggestion
                raise ValueError(f"Task.requires method returns {type(task)}. You should return luigi.Task.")
```

Simply not only gokart related task, its base class luigi.Task can be OK :)